src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java

```diff
@@ -0,0 +1,293 @@
+package uk.gov.hmcts.reform.iataskconfiguration;
+
+import org.camunda.bpm.dmn.engine.DmnDecision;
+import org.camunda.bpm.dmn.engine.DmnDecisionRuleResult;
+import org.camunda.bpm.dmn.engine.DmnDecisionTableResult;
+import org.camunda.bpm.dmn.engine.DmnEngine;
+import org.camunda.bpm.dmn.engine.DmnEngineConfiguration;
+import org.camunda.bpm.engine.variable.VariableMap;
+import org.camunda.bpm.engine.variable.impl.VariableMapImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@SuppressWarnings({"checkstyle:LineLength", "checkstyle:AbbreviationAsWordInName"})
+class CamundaTaskInitiationTest {
+
+    private DmnEngine dmnEngine;
+
+    @BeforeEach
+    void setUp() {
+        dmnEngine = DmnEngineConfiguration
+            .createDefaultDmnEngineConfiguration()
+            .buildEngine();
+    }
+
+    @DisplayName("Get task id")
+    @ParameterizedTest(name = "\"{0}\" \"{1}\" should go to \"{2}\"")
+    @CsvSource({
+        "submitAppeal, anything, processApplication, TCW, 2, false, ",
```

Comment:
"submitAppeal, anything, processApplication, TCW, 2, false, " ->should check task category for processApplication is Case Progression.

Reply:
I raised this question with Helen. Will update it accordingly.

Reply:
Helen says to leave it as it is